# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1288898585 25200
#      Thu Nov 04 12:23:05 2010 -0700
# Node ID 452dc55cc55612e2961c012365c755275b970802
# Parent  01ddaedd6ad5b6af061373cc6daa7cb2d2a94c4a
check-m-sources target in scripts directory now includes checking generated .m files.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-11-04  Rik  <octave@nomad.inbox5.com>
+
+	* Makefile.am: Update check-m-sources target to include generated
+	  function files.
+
 2010-11-04  Rik  <octave@nomad.inbox5.com>
 
 	* general/accumarray.m, general/accumdim.m, general/blkdiag.m,
 	  general/cell2mat.m, general/common_size.m,
 	  general/private/__isequal__.m, general/private/__splinen__.m,
 	  geometry/voronoi.m, io/textscan.m, miscellaneous/private/__xzip__.m,
 	  pkg/pkg.m, plot/axis.m, plot/pareto.m, plot/private/__patch__.m,
 	  statistics/base/mode.m, strings/deblank.m, strings/strcat.m,
